src/GSF.CarbonAware/src/Models/CarbonIntensity.cs

```diff
@@ -0,0 +1,32 @@
+namespace GSF.CarbonAware.Models;
+
+public record CarbonIntensity
+{
+    /// <summary>the location name where workflow is run </summary>
```

Comment:
NIT: Capitalize first letter (here and elsewhere)

Reply:
Removing this model now